Library/Formula/qbittorrent.rb

```diff
@@ -0,0 +1,36 @@
+class Qbittorrent < Formula
+  desc "Qt based free and reliable P2P Bittorrent client"
+  homepage 'http://www.qbittorrent.org'
+  url 'https://github.com/qbittorrent/qBittorrent/archive/release-3.2.3.tar.gz'
+  sha256 "816d611a0af50c469f52946c94b88c003f908ee031ee0e55a45273e4c64897c5"
+  head 'https://github.com/qbittorrent/qBittorrent.git',
+    :branch => ENV['QBT_BRANCH'] || "master"
+
+  depends_on "pkg-config" => :build
+  depends_on "qt" => :build
+  depends_on "libtorrent-rasterbar" => :build
+  depends_on "boost" => :build
+  depends_on "gnu-sed" => :build
+
+  resource "geoIP" do
+    url "http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
```

Comment:
Kind of a big deal. We warn every time if a checksum isn't attached to a resource in a formula, unless that resource is from a git checkout or such.

```
==> Downloading http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
######################################################################## 100.0%
Warning: Cannot verify integrity of qbittorrent--geoIP-3.2.3.gz
A checksum was not provided for this resource
For your reference the SHA256 is: c03b4b8e10d67626fb3cd67cb338d186ce49273be60519dc4ad2ff073437bb87
```